src/components/Card/Card.tsx

```diff
@@ -86,14 +89,15 @@ function Card({
   };
 
   return (
-    <div
+    <Link
+      to={redirectUrl ?? ''}
```

Comment:
Is there any reason the url can't be a required prop?

Reply:
I set it optional because of this `isInView` flag `url={isInView ? url : undefined}` in Shelf.tsx

Reply:
I did some investigating into what it means to set a link to a blank string, and I don't think it is going to do what you expect it to: https://stackoverflow.com/a/76738446/1345424

I'd suggest you set make url required, always set it, and just add a disabled prop when isInView is false.